src/app/prevention/strides-against/strides-against.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="row">
+  <div class="col-lg-6 col-lg-offset-3">
+    <h2 class="sub-heading-1 text-center">{{language?.heading}}</h2>
+  </div>
+</div>
+<div class="row">
+  <div class="col-md-4 col-md-offset-3">
```

Comment:
It doesn't appear centered on my system.
`<div class="col-md-4 col-md-offset-2">` this helped.
Does it look fine on yours? if not then please change the offset value from 3 to 2.

Reply:
I don't know but at 100% zoom in my system, offset-3 worked.